Automattic/jetpack
# Update/my jetpack agency banner

This PR changes the copy and CTA link on the Jetpack Manage/Agencies banner in My Jetpack.

Changes the Banner to read:

_Title:_ Manage client sites and grow your business
_Subtitle:_ Are you an agency or freelancer? We’re working on a new partnership program bringing together the best of Jetpack, Woo, [WordPress.com](http://wordpress.com/), and Pressable. Get bulk discounts, referral commissions, and more.
_CTA:_ Register your interest
**Link:** [https://automattic.com/for-agencies](https://href.li/?https://automattic.com/for-agencies)

**Screenshots:**

BEFORE:

![Screen Shot 2024-05-06 at 17 59 31](https://github.com/Automattic/jetpack/assets/11078128/87f897c8-4c0d-4930-943e-240d2cef83b0)

.

AFTER:

![Screen Shot 2024-05-06 at 17 56 29](https://github.com/Automattic/jetpack/assets/11078128/b485b52d-9096-4810-9395-a28f63929485)





Related to: https://github.com/Automattic/jetpack-marketing/issues/739


### Other information:


## Jetpack product discussion



Project Thread: pbNhbs-axp-p2


## Does this pull request change what data or activity we track or use?



No

## Testing instructions:






* On a Jurassic Ninja site with Beta plugin, running this branch (`update/my-jetpack-agency-banner`)
* Connect Jetpack.
* Go to the My Jetpack page.
* Scroll down to and view the Jetpack Manage/Agency banner. (See screenshot above).
* Verify the title and description copy matches the copy described in this PR description.
* Verify the CTA reads, "Register your interest" and when clicked it opens https://automattic.com/for-agencies
